### ISSUE TITLE
Create factories and tests for the authentication app in the backend

### ISSUE THREAD

**Issue by to-sta** [activist-org/activist#507]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

The task is to create test data (factories) with [factory_boy](https://factoryboy.readthedocs.io/en/stable/) for the models in the authentication app. 

```bash
Name                            Stmts   Miss  Cover
---------------------------------------------------
...
authentication\models.py           49      6    88%
...
```
Have a look at the `content/factory.py` and `content/tests.py` to get started.

You can run `pytest` inside backend directory and a report terminal and html report will be generated. The html report shows you excatly what needs to be tested. The html report will saved in the folder "htmlcov".

### Contribution

Happy to help someone with this task.

**Comment by IsaacDMz** [comment]
Hey, i would love to work on this task. Can assign this to me?

**Comment by andrewtavis** [comment]
Thanks for your willingness to help, @IsaacDMz! Let us know if you need any support 😊

**Comment by andrewtavis** [comment]
FYI @to-sta, I always mark `good first issue` with `help wanted` as there are some issue aggregators that parse for one or the other, but not both :) Obviously not always the other way around though 🙃 Thanks for making all these!

**Comment by andrewtavis** [comment]
Closed by #540 :) Thanks @IsaacDMz! Really appreciate the work here 😊 Feel free to drop by the dev sync on Saturday if you have time. Details for that will be in `Development` on Matrix :)
